students/Charith_Acharya1/Circle_excercise.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 13 14:05:17 2019
+
+@author: acharch
+"""
+
+
+import math
+
+class Circle:
+
+    def __init__(self, radius):
+        self.radius = radius
+        self.__area = radius ** 2 * math.pi
+
+    @property
+    def diameter(self):
+        return self.radius * 2
+
+    @diameter.setter
+    def diameter(self, value):
+        self.radius = value / 2
+
+    @property
+    def circle_area(self):
+         self.area = math.pi * self.radius * self.radius
+         return self.area 
+     
+    @classmethod
+    def circle_from_diameter(cls, diameter):
+        cls.area_dia = math.pi * diameter/2 * cls.diameter/2
+        return cls.area_dia
```

Comment:
method is supposed to be called `from_diameter` and supposed to return an instance of Circle